testdata/docker/Dockerfile.builder

```diff
@@ -0,0 +1,15 @@
+# This Dockerfile generates a build environment for generating ELFs
```

Comment:
Why is this file `Dockerfiler.builder` and not just `Dockerfile`? This means I can't build just using `docker build .`.

Reply:
How about adding
```
ENTRYPOINT ["make"]
```

as per https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#entrypoint ? It seems nice that the default invocation of the container invokes make.

Reply:
Good point re: Dockerfile name. Was originally done for clarity, but the README specifies that it's a builder image, so no longer matters. Done!

`make` entrypoint would be nice, but dropping into a shell then requires `--entrypoint /bin/bash` (and that flag still doesn't have a shorthand :angry:). If you feel strongly about this, we can change it.

Reply:
Nope, let's keep it as it is.

Reply:
@ti-mo I was about to ask the same. Dockerfiles can be named anything as long they have the suffix Dockerfile, so "builder.Dockerfile" would be acceptable :)